### ISSUE TITLE
Can we add a feature such that if the user long presses os swipes a joke towards left he gets an option to remove that specific joke container?

### ISSUE THREAD

**Issue by toruwatanbe** [iampawan/FunnyBones#4]
I am new to Flutter and open source and I found this project to be good for a beginner like me. Please feel free to share your thoughts @iampawan

**Comment by Sahilb315** [comment]
yes bro we can add this sliding option to delete a note(or anything). We can do it by both ways by long pressing it or sliding it left. I can help you on this if required

**Comment by jagrutatwork** [comment]
Hello Sahil this is Jagrut (toruwatanabe), this is my second Github account. We can definitly work on the feature you are talking about. Please ping me at jagrutatwork@gmail.com if you are willing to help me out. Thanks
